nixos/modules/services/web-servers/unit/default.nix

```diff
@@ -35,6 +35,14 @@ in {
         default = "/var/log/unit";
         description = "Unit log directory.";
       };
+      sandboxWriteDirs = mkOption {
```

Comment:
Personally I don't like this option because it does almost nothing. Just instruct the user to set `systemd.services.unit.serviceConfig.ReadWritePaths` and we have the same result.

Reply:
Here the same method is used:
https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/databases/foundationdb.nix#L305-L317
Easier to navigate in the configuration

Reply:
I also disagree with that option. It seems frivolous at best, in my opinion. We're simply copying verbatim existing options. Again, just my opinion. Others are free to disagree :man_shrugging:

Reply:
Removed